setup.py

```diff
@@ -74,5 +74,13 @@
     version = '{version}'
 """.lstrip()
 
+current_path = os.path.abspath(os.path.dirname(__file__))
+
+def read_file(*parts):
+    with open(os.path.join(current_path, *parts), encoding='utf-8') as reader:
+        return reader.read()
+
 setup(use_scm_version={'write_to': os.path.join('astrocut', 'version.py'),
-                       'write_to_template': VERSION_TEMPLATE})
+                       'write_to_template': VERSION_TEMPLATE},
+     long_description=read_file('README.rst'),
+     long_description_content_type='text/x-rst')
```

Comment:
note that this can fail if there are errors parsing the rst file. we can address if they come up later, just mentioning here so we are prepared.